.github/workflows/cron-build.yml

```diff
@@ -99,6 +99,13 @@ jobs:
               with:
                 ref: ${{ github.event.inputs.ref || 'main' }}
 
+            - name: Downgrade VirtualBox as a workaround for networking issues
+              # TODO: this is horrible. Remove this when we have a long-term solution.
```

Comment:
Need to link out a CMOS really, I think we have one somewhere. Really it is a Virtualbox or CB vagrant issue and not ours.

Reply:
https://issues.couchbase.com/browse/CMOS-135 - will add to the commit